model_server/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM --platform=linux/amd64 python:3.8 AS base
+
+# Allow statements and log messages to immediately appear in the Knative logs
+ENV PYTHONUNBUFFERED True
+
+# set a directory for the app
+WORKDIR /app
+
+# install dependencies - setuptools use git to find non python files, hence the magic invocation
+RUN apt update && apt-get install -y git-lfs
+COPY pyproject.toml pyproject.toml
+ARG PSEUDO_VERSION=1
+RUN SETUPTOOLS_SCM_PRETEND_VERSION=${PSEUDO_VERSION} pip install --no-cache-dir -e .[test]
+
+# Install the app
+COPY src ./src
+RUN pip install --no-cache-dir -e .
+
+FROM base
+
+ARG MODEL_NAME
+ARG MODEL_TYPE
+ARG DEFAULT_ACTION
+
+# Setup the configuration defaults
+RUN echo "MODEL_NAME=${MODEL_NAME}\n\
+MODEL_TYPE=${MODEL_TYPE}\n\
+DEFAULT_ACTION=${DEFAULT_ACTION}" >> .env
+
+# Fetch the appropriate model
+RUN mkdir models
+RUN git clone https://huggingface.co/${MODEL_NAME} ./models/${MODEL_NAME}
+RUN cd ./models/${MODEL_NAME} && git lfs install && git lfs pull
```

Comment:
Does this order work? I thought `git lfs install` needs to run BEFORE `git clone` with the large huggingface models.

Reply:
it seems to work... I can move it over. Or perhaps the `apt-get install` also runs `git lfs install`? Interesting

Reply:
Oh, I see you already have `apt-get install -y git-lfs` above. Then is `git lfs install` here redundant?